project/settings/defaults.py

```diff
@@ -283,3 +300,5 @@
     messages.WARNING: 'alert-warning',
     messages.ERROR: 'alert-danger',
 }
+EMBER_EXP_PLAYER_REPO = 'https://github.com/pattisdr/ember-frame-player'
```

Comment:
@pattisdr Would you work with @icereval to transfer your ember-frame-player repo to COS?